test/corpus/expressions.txt

```diff
@@ -424,3 +424,28 @@ a ? function(){ return true } : b;
           (return_statement
             (true))))
       (identifier))))
+
+==========================================================
+Class Component with content
+:language(gts)
+==========================================================
+
+export class Menu extends Component<{
+  Args: {
+    /* args here */
+  }
+}> {
+  <template>
+    {{#let (IsOpen) (TriggerElement) as |isOpen triggerEl|}}
+    {{/let}}
+  </template>
+}
+
+----
+
+(program
```

Comment:
this is wrong, but I haven't gotten to the point where I can run the tests with the gts language yet, because I haven't been able to build them yet